### ISSUE TITLE
Extension: Options page

### ISSUE THREAD

**Issue by tomusdrw** [paritytech/parity-ui#52]
Should be possible to specify:
1. Custom DApps port
2. Custom Signer port
3. DApps credential (in case basic authentication is used)

**Comment by goldylucks** [comment]
only in extension, not in signer dapp?

**Comment by tomusdrw** [comment]
Yup, I think it doesn't make much sense in a dapp (it always connects to the same port it's served and cannot configure proxy)
